vtest/tablature-1.mscx

```diff
@@ -1,28 +1,75 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<museScore version="2.07">
```

Comment:
These changes are the result of running mscore with "-t" and allowing "Automatic Placement" updates (moving from version 2) with the addition of the code update for glissandos as in previous commits